tests/lambda_endpoints/test_queries.py

```diff
@@ -15,19 +16,20 @@
 
 
 @pytest.mark.network()
+@openmock
 class TestQueries(unittest.TestCase):
     def setUp(self):
         # Opensearch client Params
         os.environ[
             "OS_DOMAIN"
-        ] = "search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-west-2.es.amazonaws.com"
+        ] = "search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-east-1.es.amazonaws.com"
```

Comment:
You could consider doing this. You would still have to hard-code the DomainName though so maybe not worth the change.

        session = boto3.Session()

        # get the opensearch client
        opensearch_client = session.client('opensearch')

        # describe the OpenSearch domain and get its endpoint
        domain_description = opensearch_client.describe_domain(DomainName='sdsmetadatadomain-tlcs-dev')
        os.environ["OS_DOMAIN"] = domain_description['DomainStatus']['Endpoint']

Reply:
That is a really good idea @laspsandoval! Right now this test doesn't even pass I don't think because it requires the opensearch stack to be setup and running already, so we are testing against deployed resources. @sdhoyt is that right, or is there a different way to test this?

Also, I just found out about `vcrpy` that may be an interesting thing to investigate for these remote resources and testing against them without requiring them to be available 24/7.
https://vcrpy.readthedocs.io/en/latest/
with a pytest plugin that we could make use of:
https://pytest-vcr.readthedocs.io/en/latest/
For another time though...